packages/dhis2/src/Adaptor.js

```diff
@@ -472,6 +513,56 @@ export function get(resourceType, query, options = {}, callback = false) {
   };
 }
 
+/**
+ * Post data. Generic helper method for posting data of any kind to DHIS2.
+ * - This can be used to create `DataValueSets`,`events`,`trackers`,`etc.`
```

Comment:
Please remove the `-` and indent at the start of this line. Please also remove the backticks around `etc`